app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -306,8 +306,11 @@ function handleFocus(event) {
     area.blur();
   } else {
     var form = event.target.closest(".comment-form");
-    form.classList.add("comment-form--initiated");
+    if (form) {
+      form.classList.add("comment-form--initiated");
+    }
     handleSizeChange(event);
+    window.Forem.initializeMentionAutocompleteTextArea(area);
```

Comment:
Love that this means that we only load the components when a person starts typing ❤️ 
I see on [bundlephobia](https://bundlephobia.com/result?p=@reach/combobox@0.13.2) that it adds 7.9kb

Is this the same for the notifications page?

Reply:
it is - the notifications page comment area uses this same code 👍